janusgraph-cql/pom.xml

```diff
@@ -71,7 +71,6 @@
         <dependency>
             <groupId>org.apache.cassandra</groupId>
             <artifactId>cassandra-all</artifactId>
-            <optional>true</optional>
```

Comment:
This is needed otherwise Spark traversal does not work for CQL. I checked 0.5.3's distribution and saw cassandra-all.jar under lib folder. My guess is it was included by some other libraries as a transitive dependency.

Reply:
In v0.5, `cassandra-all` was introduced by `janusgraph-cassandra` module, which is removed from v0.6. Therefore we need to make this non-optional to make Spark traversal work.